db/schema.rb

```diff
@@ -323,6 +323,9 @@
     t.integer  "creator_id",         limit: 4,   null: false
     t.datetime "created_at",                     null: false
     t.datetime "updated_at",                     null: false
+    t.integer  "deploy_group_id",    limit: 4,   null: false
```

Comment:
then don't :)

On Thu, May 12, 2016 at 9:33 PM, ian waters <notifications@github.com>
wrote:

> In db/schema.rb
> <https://github.com/zendesk/samson/pull/977#discussion_r63132455>:
>
> > @@ -323,6 +323,9 @@
> >      t.integer  "creator_id",         limit: 4,   null: false
> >      t.datetime "created_at",                     null: false
> >      t.datetime "updated_at",                     null: false
> > +    t.integer  "deploy_group_id",    limit: 4,   null: false
>
> so this is a good question. there is no real reason to put these in there
> as long as we don't mind parsing the key to find secrets that relate to a
> given set of objects. thoughts?
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/977/files/c3219e2e5083523e7a77314738c623a001a7f655#r63132455>
>